tests/queries.rs

```diff
@@ -75,25 +109,37 @@ fn run_insert_query(tx: &mut Transaction, query: &str) {
 }
 
 async fn run_match_aggregate_query(tx: &mut Transaction, query: &str) -> Numeric {
-    tx.query.match_aggregate(query).await.unwrap_or_else(|err| panic!("An error occurred running a Match Aggregate query: {}", err))
+    tx.query
+        .match_aggregate(query)
+        .await
+        .unwrap_or_else(|err| panic!("An error occurred running a Match Aggregate query: {}", err))
 }
 
 #[tokio::test(flavor = "multi_thread")]
 #[ignore]
 async fn basic() {
     let mut client = new_typedb_client().await;
     create_db_grakn(&mut client).await;
-    println!("{}", client.databases.all().await
-        .unwrap_or_else(|err| panic!("An error occurred listing databases: {}", err))
-        .iter().fold(String::new(), |acc, db| acc + db.name.as_str() + ",")
+    println!(
+        "{}",
+        client
+            .databases
+            .all()
+            .await
+            .unwrap_or_else(|err| panic!("An error occurred listing databases: {}", err))
+            .iter()
+            .fold(String::new(), |acc, db| acc + db.name.as_str() + ",")
```

Comment:
Putting every chained call on a new line is definitely ugly in cases like these - can we tell `rustfmt` to not do this?

Reply:
That does not appear to be an option.